opensoundscape/seed.py

```diff
@@ -0,0 +1,15 @@
+""" seed.py: Set random state across different libraries for reproducibility
+"""
+import numpy as np
+import torch
+import random
+
+
+def seed(seed, verbose=True):
```

Comment:
needs a docstring, with Args and Return fields; see other functions for examples

Reply:
Done